src/wizard/deploy.jl

```diff
@@ -25,7 +25,7 @@ function print_build_tarballs(io::IO, state::WizardState)
         """
     end
 
-    if state.files === nothing
+    if state.files === nothing || state.file_varnames === nothing
         products_string = "Product[\n]"
```

Comment:
I'd check that the zip below isn't nothing

Reply:
zip itself does not return nothing:
```
julia> zip([], nothing)
zip(Any[], nothing)

julia> collect(zip([], nothing))
ERROR: MethodError: no method matching length(::Nothing)
```

Reply:
:disappointed:

Reply:
you mean the `state.files`?

Reply:
My worry is that `state.file_kinds` can be `nothing` and we again don't catch it

Reply:
done

Reply:
Done what? :thinking:

Reply:
well, it was done then. Now it is pushed :see_no_evil:

Reply:
Just realised I commented on the wrong line :weary: